build/configs/cy4390x/tc/defconfig

```diff
@@ -1076,6 +1075,8 @@ CONFIG_KERNEL_CMDS=y
 CONFIG_FS_CMDS=y
 CONFIG_FSCMD_BUFFER_LEN=64
 CONFIG_NET_CMDS=y
+CONFIG_NET_PING_CMD=y
```

Comment:
Please exclude something which is not related to procfs from this commit.

Reply:
I modified it through Kconfig without modifying defconfig directly.
File Systems -> Auto mount The PROCFS File System was turned on,
The result is another change.
Is it correct to edit only the auto mount by hand?

Reply:
modify!